draft-ietf-scitt-architecture.md

```diff
@@ -388,32 +388,34 @@ Registration Policies refer to additional checks over and above the Mandatory Re
+For instance, in order to authenticate X.509 Signed Statements, the Transparency Service MUST build and validate a complete certificate chain from the Issuer's certificate identified by `x5t`, to one of the root certificates most recently registered as a trust anchor of the Transparency Service.
+
+The Transparency Service MUST evaluate the Registration Policy that was most recently added to the Append-only Log.
 
-- Authenticate the Issuer of the Signed Statement
+#### Auditability of Registration
 
-The Transparency Service MUST authenticate the Issuer of Signed Statements by validating the COSE signature and checking the identity of the issuer through one of its configured trust anchors, using the `x5t` and `kid` headers in the protected header as hints. For instance, for X.509 signed claims the Transparency Service must validate a complete certificate chain from the certificate identified by `x5t` to one of the trusted root authority certificate of the Transparency Service.
-The public key is used to verify digital signatures, and the associated data is used to constrain the types of information for which the trust anchor is authoritative."
+The operator of a Transparency Service MAY update the Registration Policy or the trust anchors of a Transparency Service at any time.
 
+Transparency Services MUST ensure that for any Transparent Statment they accept, enough information is made available to Auditors (either in the Append-only Log and retrievable through audit APIs, or included in the Receipt) to identify and recover the Transparent Statements describing the registration policy and trust anchors that are applicable to the Transparent Statement.
 
-Before a Signed Statement is registered, the trust anchor used to verify its Issuer MUST be registered with the Transparency Service.
 
 ### Initialization and bootstrapping {#ts-initialization}
 
-Since a Registration Policy is required prior to the registration of any Signed Statements, a means is required to configure the first Registration Policy that is not the standard issuance of a Signed Statement.
-Transparency Services MUST support at least one of these methods:
+Since the mandatory registration checks rely on having registered Signed Statements for the registration policy and trust anchors, Transparency Services MUST support at least one of the three following bootstrapping mechanisms:
 
-- A built-in default Registration Policy
-- Acceptance of a first Signed Statement whose payload is a valid Registration Policy, without performing registration checks
-- An out-of-band authenticated management interface
+- A built-in default Registration Policy and initial list of trust anchors;
```

Comment:
Suggestion to revert this line
```suggestion
- A built-in default Registration Policy
```

Reply:
looking for feedback with a +1 or -1 to merge or discard

Reply:
+1 on reverting

Reply:
-1 the trust anchors are as important (in fact probably more important) than the registration policy for trusting the TS. Imagine the operator adds his own self-signed root CA suddenly, then he can impersonate any issuer

Reply:
+1 revert. A TS impersonating an  issuer sounds like an attack to me, so why would I trust any anchors it self registers ?